packages/pluggableWidgets/rich-text-web/src/utils/ckeditorPresets.ts

```diff
@@ -0,0 +1,146 @@
+import { CKEditorConfig } from "ckeditor4-react";
+
+export interface ToolbarGroup {
+    name: string;
+    groups?: string[];
+}
+
+export interface ToolbarItems {
+    name: string;
+    items: string[];
+}
+
+export type GroupType =
+    | "documentGroup"
+    | "formsGroup"
+    | "editingGroup"
+    | "clipboardGroup"
+    | "basicStylesGroup"
+    | "paragraphGroup"
+    | "linksGroup"
+    | "colorsGroup"
+    | "toolsGroup"
+    | "othersGroup"
+    | "stylesGroup";
+
+const TOOLBAR_GROUP: ToolbarGroup[] = [
+    { name: "document", groups: ["mode", "document", "doctools"] },
+    { name: "clipboard", groups: ["clipboard", "undo"] },
+    { name: "editing", groups: ["find", "selection", "spellchecker", "editing"] },
+    { name: "forms", groups: ["forms"] },
+    { name: "basicstyles", groups: ["basicstyles", "cleanup"] },
+    { name: "paragraph", groups: ["list", "indent", "blocks", "align", "bidi", "paragraph"] },
+    { name: "links", groups: ["links"] },
+    { name: "insert", groups: ["insert"] },
+    { name: "styles", groups: ["styles"] },
+    { name: "colors", groups: ["colors"] },
+    { name: "tools", groups: ["tools"] },
+    { name: "others", groups: ["others"] },
+    { name: "about", groups: ["about"] }
+];
+
+const TOOLBAR_ITEMS: ToolbarItems[] = [
+    {
+        name: "document",
+        items: ["Source", "-", "Save", "NewPage" /* "ExportPdf"*/, "Preview", "Print", "-", "Templates"]
+    },
+    { name: "clipboard", items: ["Cut", "Copy", "Paste", "PasteText", "PasteFromWord", "-", "Undo", "Redo"] },
+    { name: "editing", items: ["Find", "Replace", "-", "SelectAll", "-", "Scayt"] },
+    {
+        name: "forms",
+        items: ["Form", "Checkbox", "Radio", "TextField", "Textarea", "Select", "Button", "ImageButton", "HiddenField"]
+    },
+    {
+        name: "basicstyles",
+        items: ["Bold", "Italic", "Underline", "Strike", "Subscript", "Superscript"]
+    },
+    {
+        name: "paragraph",
+        items: [
+            "NumberedList",
+            "BulletedList",
+            "-",
+            "Outdent",
+            "Indent",
+            "-",
+            "Blockquote",
+            "CreateDiv",
+            "-",
+            "JustifyLeft",
+            "JustifyCenter",
+            "JustifyRight",
+            "JustifyBlock",
+            "-",
+            "BidiLtr",
+            "BidiRtl",
+            "Language"
+        ]
+    },
+    { name: "links", items: ["Link", "Unlink", "Anchor"] },
+    { name: "insert", items: ["Image", "Table", "HorizontalRule", "Smiley", "SpecialChar", "PageBreak", "Iframe"] },
+    { name: "styles", items: ["Styles", "Format", "Font", "FontSize"] },
+    { name: "colors", items: ["TextColor", "BGColor"] },
+    { name: "tools", items: ["Maximize", "ShowBlocks"] },
+    { name: "about", items: ["About"] }
+];
+
+function SET_PRESET(type: "basic" | "standard" | "full"): CKEditorConfig {
```

Comment:
NIT: I think it's bit confusing to see function name as upper case. I agree that this is "constant" value, but I think it's better to rename it to `setPreset`